### ISSUE TITLE
Addressable::URI::InvalidURIError: Absolute URI missing hierarchical segment: 'http://'

### ISSUE THREAD

**Issue by chutterbox** [jaimeiniesta/metainspector#145]
to reproduce: 
https://metainspectordemo.herokuapp.com/scrape?url=http%3A%2F%2Fcen.acs.org%2Farticles%2F93%2Fi35%2FFirst-Metal-Organic-Framework-Made.html

(url is: http://cen.acs.org/articles/93/i35/First-Metal-Organic-Framework-Made.html)

not sure if this is related to #78

**Comment by jaimeiniesta** [comment]
Thanks for the report, this was recently fixed on 4.6.1 (but I had not updated the demo server, it's done now).

Duplicate of #144 

The reason for the exception was this line on the source of that page, look at the href value:

```html
<link rel="alternate" type="application/rss+xml" title="CEN RSS Feed" href="http://"/>
```

**Comment by chutterbox** [comment]
Awesome, thank you. This is a great gem, thanks a lot for the reply!
